applications/rogue/main.tf

```diff
@@ -108,6 +103,14 @@ locals {
     DS_ENABLE_BLINK        = var.environment != "development" # TODO: Remove after updating app to read 'DS_ENABLE_CUSTOMER_IO'.
     DS_ENABLE_CUSTOMER_IO  = var.environment != "development"
     DS_ENABLE_GAMBIT_RELAY = var.gambit_url != null
+
+    # S3 settings:
+    STORAGE_DRIVER    = "s3"
+    FILESYSTEM_DRIVER = "s3"
+    S3_BUCKET         = var.name
+    AWS_S3_BUCKET     = var.name
+    S3_REGION         = "us-east-1"
+    AWS_S3_REGION     = "us-east-1"
```

Comment:
Since we're removing the S3 bucket module, which used to provide these, I've hard-coded them under `extra_config_vars` for now to make everything continues to function exactly the same way.